theme/static/style/style.css

```diff
@@ -413,6 +413,23 @@ pre code {
     cursor: pointer;
 }
 
+/* Seen At Section */
+
+.fosdem-img {
+    width: 200px;
+    height: 200px;
+    background-image: url('../../images/conf/fosdem.png');
+    background-color: inherit;
+    background-repeat: no-repeat;
+    display: inline-block;
+    float: left;
```

Comment:
This div is out of alignment with the rest of the images. Use `margin-left: 1.5rem;`